### ISSUE TITLE
Upstream plugin to mineflayer 

### ISSUE THREAD

**Issue by rom1504** [IceTank/mineflayer-item-map-downloader#8]
I think this is low level enough this should just be part of mineflayer

**Comment by IceTank** [comment]
I can look into it. But note that this implementation is not complete. It can only construct maps from single map packets that act in a map initialization way. Any updates to the map in subsequent packets are discarded. The minecraft protocol supports multiple ways to update map data. Including ways were only some sections like rectangles or stripes are updated instead of the entire map. Labels are also currently not supported.

**Comment by rom1504** [comment]
Yeah I think we should support all that. Doesn't seem very hard no?
